<launch>
  <!--node pkg="mslquad"
        type="se3controller"
        name="se3controller"
        output="screen">
    <param name="KP" value="4.0" />
    <param name="KV" value="6.0" />
    <param name="KR" value="0.3" />
    <param name="KW" value="0.05" />
    <param name="M" value="1.04" />
    <param name="TCOEFF" value="4.4" />
    <param name="MODEL" value="mslquad"/>
  </node-->



    <include file="$(find mavros)/launch/px4.launch">
      <arg name="fcu_url" value="udp://:14540@127.0.0.1:14557" />
    </include>

    <param name="use_sim_clock" type="bool" value="true" />


    <node pkg="asl_flight" type="asl_traj_ctrl" name="asl_traj_ctrl" output="screen" >
      <param name="KP" value="4.0" />
      <param name="KV" value="4.0" />
      <param name="KR" value="2.0" />
      <param name="KW" value="0.5" />
      <param name="M" value="1.5" />
      <param name="TRAJ" type="str" value="CIRCLE" />
      <param name="TCOEFF" value="6.57" />
      <param name="MODEL" value="iris"/>
    </node>

    <!-- param name="use_sim_clock" type="bool" value="true" -->

</launch>
